### PR TITLE
Creating Process models and a Processes collection

### PR DESCRIPTION
**Pull request by isuftin**

### CODE REVIEW

File: src/main/webapp/js/advanced/collections/Processes.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*jslint browser: true*/
+/*global Backbone*/
+var GDP = GDP || {};
+
+GDP.collection = GDP.collection || {};
```
Comment (mbucknell) [inline src/main/webapp/js/advanced/collections/Processes.js]: I think want to namespace it as GDP.Advanced.collection.

File: src/main/webapp/js/advanced/models/Process.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*jslint browser: true*/
+/*global Backbone*/
+var GDP = GDP || {};
+
+GDP.model = GDP.model || {};
+
+GDP.model.Process = Backbone.Model.extend({});
```
Comment (mbucknell) [inline src/main/webapp/js/advanced/models/Process.js]: Same comment about namespacing here.

File: src/main/webapp/js/advanced/collections/Processes.js (hunk 1 above)
Comment (isuftin) [inline src/main/webapp/js/advanced/collections/Processes.js]: ok. is it GDP.ADVANCED, GDP.Advanced or GDP.advanced?

Comment (isuftin) [inline src/main/webapp/js/advanced/collections/Processes.js]: Looks like ADVANCED

**Pull request closed without merging**